airbyte-integrations/connectors/destination-s3/src/test/java/io/airbyte/integrations/destination/s3/AesCbcEnvelopeEncryptionBlobDecoratorTest.java

```diff
@@ -0,0 +1,73 @@
+package io.airbyte.integrations.destination.s3;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.Charset;
+import java.util.Base64;
+import java.util.Base64.Decoder;
+import java.util.HashMap;
+import java.util.Map;
+import javax.crypto.spec.SecretKeySpec;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class AesCbcEnvelopeEncryptionBlobDecoratorTest {
+
+  private static final Decoder BASE64_DECODER = Base64.getDecoder();
+
+  private AesCbcEnvelopeEncryptionBlobDecorator decorator;
+
+  @BeforeEach
+  public void setup() {
+    decorator = new AesCbcEnvelopeEncryptionBlobDecorator(
+        new SecretKeySpec(BASE64_DECODER.decode("oFf0LY0Zae9ksNZsPSJG8ZLGRRBUUhitaPKWRPPKTvM="), "AES"),
+        new SecretKeySpec(BASE64_DECODER.decode("9ZAVuZE8L4hJCFQS49OMNeFRGTCBUHAFOgkW3iZkOq8="), "AES"),
+        BASE64_DECODER.decode("04YDvMCXpvTb2ilggLbDJQ==")
+    );
+  }
+
+  @Test
+  public void testEncryption() throws IOException {
+    final ByteArrayOutputStream stream = new ByteArrayOutputStream();
+
+    try (final OutputStream wrapped = decorator.wrap(stream)) {
+      IOUtils.write(
+          """
+              adc66b6e-6051-42db-b683-d978a51c3c02,"{""campaign.resource_name"":""cus""}",2022-04-04 22:32:50.046
+              0e253b28-bec6-4a90-8622-629d3e542982,"{""campaign.resource_name"":""cus""}",2022-04-04 22:32:50.047
+              """,
+          wrapped,
+          Charset.defaultCharset()
```

Comment:
Better to use a specific charset than the default one, because the default on depends on the JVM, and the variance may introduce some uncertainty. The gradle FindBugs plugin usually complains about this.

```suggestion
          StandardCharsets.UTF_8
```

Reply:
done (surprisingly, `:airbyte-integrations:connectors:destination-s3:spotbugsTest` doesn't actually catch this)